src/v2/view-builder/views/shared/IdentifierFooter.js

```diff
@@ -0,0 +1,51 @@
+import { loc } from 'okta';
+import BaseFooter from '../../internals/BaseFooter';
+import { FORMS as RemediationForms } from '../../../ion/RemediationConstants';
+import { getForgotPasswordLink } from '../../utils/LinksUtil';
+
+export default BaseFooter.extend({
```

Comment:
Need a spec file for this ?

Reply:
Im leading toward to be covered when testing first class Views instead of individual spec for this shared component.